pkg/http/core/kubernetes/deploy_test.go

```diff
@@ -209,6 +209,42 @@ var _ = Describe("Deploy", func() {
 		})
 	})
 
+	When("The manifest is versioned", func() {
+		BeforeEach(func() {
+			fakeKubeController.IsVersionedReturns(true)
+		})
+
+		When("Listing resources by kind and namespace returns an error", func() {
+			BeforeEach(func() {
+				fakeKubeClient.ListResourcesByKindAndNamespaceReturns(nil, errors.New("ListResourcesByKindAndNamespaceReturns fake error"))
+			})
+
+			It("ListResourcesByKindAndNamespace returns a fake error", func() {
+				Expect(c.Errors.Last().Error()).To(Equal("ListResourcesByKindAndNamespaceReturns fake error"))
```

Comment:
Add an expect case for the response status code returned. Example [here](https://github.com/homedepot/go-clouddriver/blob/671fe4b0dcb25adb5b85ef6fa775e1a6d4a8ce19/pkg/http/core/kubernetes/deploy_test.go#L207).

Reply:
done